hybrasyl/CreatureStatus.cs

```diff
@@ -105,11 +118,13 @@ public class CreatureStatus : ICreatureStatus
         public string UseCastRestrictions => XMLStatus.CastRestriction?.Use ?? string.Empty;
         public string ReceiveCastRestrictions => XMLStatus.CastRestriction?.Receive ?? string.Empty;
 
+        public StatusInfo Info => new StatusInfo() { Name = Name, CastableName = CastableName, Remaining = Remaining };
```

Comment:
CastableName should not be applicable to a status.

Reply:
See above.